Aztec/Classes/Private/Libxml2/Data/ElementNode.swift

```diff
@@ -449,6 +454,43 @@ extension Libxml2 {
             return self
         }
 
+        /// Returns the lowest-level text node in this node's hierarchy that wraps the specified
+        /// range.  If no child element node wraps the specified range, this method returns nil.
+        ///
+        /// - Parameters:
+        ///     - range: the range we want to find the wrapping node of.
+        ///
+        /// - Returns: the lowest-level text node wrapping the specified range, or nil if
+        ///         no child node fulfills the condition.
+        ///
+        func lowestTextNodeWrapping(range: NSRange) -> TextNode? {
```

Comment:
Could you add a few unit tests for this method?

I'd like to make sure any future changes won't break this.

Ideally:
- At least one for a good result.
- At least one for a nil result.

`ElementNodeTests.swift` has a lot of examples you can use.  Please let me know.